### PR TITLE
Wins page modal a11y

### LINKED ISSUE #1531: Make wins' page modal overlay keyboard navigable
### Overview
Individual wins' card and wins' page modal overlay is not keyboard accessible.

### Action Items
- [x] make "...see more" tab-able 
- [ ] focus on modal overlay when displayed
- [x] make escape available to modal overlay  

### Resources/Instructions
[Accessible Links](https://www.htmldog.com/guides/html/advanced/links/)
[Creating an Accessible Modal Dialog](https://bitsofco.de/accessible-modal-dialog/)

### PR DESCRIPTION
**Pull request by grhhhh**
fixes #1531

-Makes "see more..." tab-able. 
-Makes modal close when `ESC` key is pressed. 
-Captures focus so that modal content is tab-able and background content is ignored while the modal is open.

Note: this fix requires the use of the inert polyfill

### CODE REVIEW

**Comment by Aveline-art** [comment]
@grhhhh I can't seem to tab-focus inside the modal. Am I doing it right? This is what I have tried.

1. Click 'see more' to open modal.
2. Press tab.
3. Tab scrolls focus over 'linkedin' and 'github' links.

<details>
<summary>Am I supposed to tab when the state of the page is like this?</summary>

</details>

For specs, I use the latest version of Firefox on Windows.

**Comment by grhhhh** [comment]
@Aveline-art can you be specific regarding which 'linkedin' and 'github' links are focused by tab as you described? Specifically whether tab focuses on the links inside of the modal or outside of it.

The intended function and the functionality that I observed (macbook pro, chrome) is that tab focuses on the modal content, in this case the 'linkedin' and 'github' links, while ignoring all things outside of the modal.

**Comment by Aveline-art** [comment]
@grhhhh Sorry for not being more clear. To clarify: when I press tab, the focus scrolls to the links inside the modal. Once those options are exhausted, the focus returns to behind the modal. At a certain point the focus cycles back to the modal, before continuing behind the modal, and so on and so on.

File: pages/wins/wins.html [hunk 1]
```diff
@@ -18,8 +18,13 @@
 	</div>
 </section>
 
+
+<script src="/node_modules/wicg-inert/dist/inert.min.js"></script>
```
Comment (akibrhast) [inline pages/wins/wins.html]: I noticed that you have added a node-modules in our project folder in this PR. And you are importing the package by referencing from the node-modules folder. Which in turns results in the addition of the package-lock.json file.

Could you please remove the node-modules folder, and the package-lock.json and instead download the inert.min.js package directly and place in the `assets/js` directory.
Such that the path of the package becomes `assets/js/inert.min.js` , and you can then modify this line to

<script src="/assets/min/inert.min.js"></script>

File: pages/wins/wins.html [hunk 2]
```diff
@@ -384,8 +389,26 @@
 		icon.setAttribute("href", href);
 		let iconImg = makeElement('img', icon, className);
 		iconImg.setAttribute("src", src);
+		// icon.setAttribute("tabindex", 0);
```
Comment (akibrhast) [inline pages/wins/wins.html]: Remove this commented line if not needed

File: pages/wins/wins.html [hunk 3]
```diff
@@ -384,8 +389,26 @@
...
 	}
   function updateOverlay(i) {
+	
+	// const modal = document.querySelector(".modal-container");
+	// console.log(modal);
```
Comment (akibrhast) [inline pages/wins/wins.html]: Remove these commented line if not needed

File: _sass/components/_wins-page.scss [hunk 4]
```diff
@@ -1,3 +1,13 @@
+// .project-card.wins-card.center-screen {
+//     position: fixed;
+//     z-index: 1;
+// }
+
+// .overlay.display-initial {
+//     //position: fixed;
+//     //z-index: -1;
+// }
+
```
Comment (akibrhast) [inline _sass/components/_wins-page.scss]: Remove commented lines if not needed.

**Review by akibrhast — CHANGES_REQUESTED** [review]

**Comment by grhhhh** [comment]
@akibrhast 
Latest commit removes package-lock.json and commented code as requested. Also, inert.min.js was added to `js/assets` directory and linked in wins.html

@Aveline-art 
I was unable to recreate the issues you described using the latest version of Firefox on my machine.

**Review by Aveline-art — APPROVED** [review]
Update June 8th, 2021. I have discovered an error on my end that was causing the problems. @grhhhh Everything is good. Great job on this!

**Comment by akibrhast** [comment]
@grhhhh Hey, were you able to resolve that issue in regards to the see more, see less button that occurs at a certain screen width.... ?

**Comment by grhhhh** [comment]
A bug was discovered while working on this issue: the ‘see more’ link changes to read ‘see less’ when opened, but does not change back to ‘see more’ after being closed. A fix falls outside of the scope of this PR.

**Review by akibrhast — APPROVED** [review]
lgtm

**Pull request merged**